triggers/service/auth/src/test/scala/com/daml/auth/oauth2/test/server/Test.scala

```diff
@@ -94,8 +108,8 @@ class Test extends AsyncWordSpec with TestFixture with SuiteResourceManagementAr
                 e => Future.failed(new IllegalArgumentException(e.toString)),
                 Future.successful(_),
               )
-            payload <- Future.fromTry(AuthServiceJWTCodec.readFromString(decodedJwt.payload))
-          } yield Right((payload, refreshToken))
+            payload <- Future.fromTry(readCustomDamlJWTTokenFromString(decodedJwt.payload))
+          } yield Right(payload, refreshToken)
```

Comment:
While I guess adapted-args are OK now, I'd leave them alone.

Reply:
Should I change it back?

Reply:
Not if it builds and is ready to merge, otherwise, please.